oci/mounts_freebsd.go

```diff
@@ -0,0 +1,38 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package oci
+
+import (
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func defaultMounts() []specs.Mount {
+	return []specs.Mount{
+		{
+			Destination: "/dev",
+			Type:        "devfs",
+			Source:      "devfs",
+			Options:     []string{},
```

Comment:
I've been using `ruleset=4` in runj so far.  A somewhat more-restrictive default might be useful here and the `jail(8)` manual page suggests using 4.

Reply:
It has been my intention to to pass devfs rule options in a (yet to be defined) OCI specification for FreeBSD. 
ruleset=4 sounds like a safe default for the meantime though :)

Reply:
It's not completely relevant for this PR, but I'm curious to know why you are thinking of embedding devfs rule options in a FreeBSD section rather than mount options.  Were you planning to allow for full ruleset definitions rather than referencing existing rulesets?

Reply:
(Note: dismissing previous LGTM to track the addition of the default ruleset option prior to merging.)

Reply:
Yes allowing to build a complete devfs ruleset has been the intention. 
FreeBSD allows the ruleset to be specified at mount time, but also allows later modifications of a devfs mountpoint via the CLI devfs tool, even changing the ruleset (rulesets in essence basically just being a series of show and hide commands).
So I'd think having a node in a FreeBSD OCI spec that allows specification of a ruleset & a list of actions to take on the dev mountpoint would be usefull.